apps/system/js/icc_worker.js

```diff
@@ -556,12 +556,12 @@ var icc_worker = {
   '0x28': function STK_CMD_SET_UP_IDLE_MODE_TEXT(message) {
     DUMP('STK_CMD_SET_UP_IDLE_MODE_TEXT:', message.command.options);
     var options = message.command.options;
-    this.idleTextNotifications[message.iccId] = new Notification(
+    this.idleTextNotifications[message.iccId] = NotificationHelper.send(
       'SIM ' + icc.getSIMNumber(message.iccId) + ' STK', {
-        body: options.text,
-        icon: 'style/icons/system.png',
-        tag: 'stkNotification_' + message.iccId,
-        mozbehavior: {
+        bodyL10n: options.text,
+        iconL10n: 'style/icons/system.png',
+        tagL10n: 'stkNotification_' + message.iccId,
+        mozbehaviorL10n: {
```

Comment:
Umm, we don't have iconL10n, tagL10n or mozbehaviorL10n and options.text is probably not an l10nId. Because this one uses mozbehavior which is not covered by NotificationHelper I'd prefer to leave it untouched.